docs/dev/building-on-zksync/contracts/contract-development.md

```diff
@@ -0,0 +1,41 @@
+# Smart contract development
+
+zkSync Era allows developers to build projects using the same programming languages and tools as required by Ethereum.
+We provide a customized and optimized compiler toolchain for development on zkSync. For more information, see
+[the toolchain documentation](../../compiler-toolchain/overview.md).
+
+## EVM compatibility
+
+zkSync Era supports almost every smart contract written for EVM and holds all key security invariants so
+that no additional security re-auditing is usually required. Notable exceptions are contracts using
+the following EVM opcodes:
+
+- `SELFDESTRUCT` - considered harmful and deprecated in [EIP-6049](https://eips.ethereum.org/EIPS/eip-6049).
+- `CALLCODE` - deprecated in [EIP-2488](https://eips.ethereum.org/EIPS/eip-2488) in favor of `DELEGATECALL`.
+- `EXTCODECOPY` - skipped at the time of writing because zkEVM opcodes are not identical in the EVM; can be implemented if needed.
+- `CODECOPY` - replaced with `CALLDATACOPY` in the deploy code.
+- `PC` - inaccessible in Yul and Solidity `>=0.7.0`; accessible in Solidity `0.6` although it produces a runtime error.
+
+::: warning
+These opcodes produce an error on compilation.
+:::
+
+There are several other distinctions, for example:
```

Comment:
I’d suggest “differences” rather than “distinctions”.  (Without a complement, “distinction” is either a positive value judgement or at a higher level.)

Reply:
Good call, I agree.

Reply:
Please do in your PR.

Reply:
191a58b3